man/deepsea-stage-0.7

```diff
@@ -0,0 +1,138 @@
+.TH deepsea 7
+.SH NAME
+deepsea stage 0 \- a deeper insight
+.SH DESCRIPTION
+.B DeepSea Stage 0 (Prep)
+is a Salt orchestration and exists mainly for cluster maintenance and operational/preparation tasks. 
+.RE
+.PD
+.SH Preface
+.PP
+Stage.0 behaves differently depending on the presence of a ceph cluster. On the first invocation, or the install phase, deepsea will run all operations in parallel to speep up the process.
+Once you successfully deployed the cluster more caution is exercised and nodes are processed serially. The operation is chunked in three sections.
+.SS Master
+
+.B Validation:
+.RS
+Deepsea performs a set of system validation tests before running the ramainder of the stage.
+
+* Deepsea minion
+  See deepsea-minions (7)
+
+* master minion
+  Check for the presence of a master minion
+
+* ceph version
+  Validate if a ceph version could be found for installation
+.RE
+
+.B Salt-Api
+.RS
+In order for openattic to establish a connection with salt, hence deepsea, the salt-api needs to be enabled.
```

Comment:
Maybe rewrite the sentence like this? "Since openATTIC communicates with DeepSea via Salt, the salt-api service must be enabled and running for openATTIC to establish a connection with Salt."